README.md

```diff
@@ -400,6 +422,14 @@ You can specify a fixed, valid value to replace a target FHIR field. For example
 For complex data types, you can provide a fixed json fragment following the [sample rules](#Sample-rules-using-FHIRPath).
 You should provide valid value for the target data type to avoid unexpected errors.
 
+## Perturb method
+With perturbation rule, you can replace specific values with equally specific, but different values. You can choose to sample noise from a fixed range or a proportional range. There are a few parameters that can help you customize the noise amount for different FHIR types.
+- [required] **span** A non-negative value representing the random noise range. For *fixed* range type, the noise will be sampled from a uniform distribution over [-*span*, *span*]. For *proportional* range type, the noise will be sampled from a uniform distribution over [-*span* * value, *span* * value]. 

```

Comment:
Mathematically, span is defined as (max - min) of the range. So, if the user specifies span, the distribution will be [-span/2 , span/2]

Reply:
Thanks, updated